backend/fms_core/services/process_measurement.py

```diff
@@ -9,15 +9,25 @@ def create_process_measurement(process=None, source_sample=None, execution_date=
     errors = []
     warnings = []
 
-    try:
-        process_measurement = ProcessMeasurement.objects.create(
-            process=process,
-            source_sample=source_sample,
-            execution_date=execution_date,
-            volume_used=volume_used,
-            comment=comment or (f"Automatically generated on {datetime.utcnow().isoformat()}Z"),
-        )
-    except ValidationError as e:
-        errors.append(f"Could not create ProcessMeasurement with Process ID {process.id} and Source sample {source_sample}")
+    # Validate parameters
+    if not process:
+        errors.append(f"Process is required for process measurement creation.")
+    if not source_sample:
+        errors.append(f"Source sample is required for process measurement creation.")
+    if not execution_date:
+        errors.append(f"Execution date is required for process measurement creation.")
+
```

Comment:
This might be useful for the devs to know if there is any problem with the code, but I doubt this information would be any helpful to the user if displayed in the front-end in the preview table